src/tests/ftest/fault_domain/fault_domain.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from apricot import TestWithServers, skipForTicket
+
+
+class FaultDomain(TestWithServers):
+    """Test class Fault domains
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up fault domain test."""
+        self.start_agents_once = False
+        self.start_servers_once = False
+        self.setup_start_agents = False
+        self.setup_start_servers = False
+        super().setUp()
+
+    @skipForTicket("DAOS-7919")
+    def test_pools_in_different_domains(self):
+        """The aims to:
+            Be able to configure daos servers using different fault domains.
+            Created pools must be in different fault domains.
+
+           Not coded, should be available when DAOS-7919 is resolved.
+           If a pool has the same fault domains, a debug message
+             should be expected.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,large
```

Comment:
Do we really need large hardware cluster? Can we achieve the same on VMs?

Reply:
That is a good question, it may be possible to use smaller pools with using only RAM, who many gigs of RAM do we have on each VM?

Reply:
Each VM should have 12G (as I am told). So yeah we can reduce the pool sizes here and move this test to VM from hardware.

Reply:
Ok, testing this

Reply:
Took me like 10 hours due to a problem with a newer version of spdk, anyways... yes, it worked using only vms.